_docs/01.2-why_use_electrode.markdown

```diff
@@ -4,14 +4,50 @@ permalink: docs/why_use_electrode.html
 
-### Learn More
+At @WalmartLabs, we recently migrated our web site to a universal JavaScript
+stack using React/Node.js, and along the way we hit a few stumbling blocks that
+might sound familiar if you've worked on a large, complex web app before.
 
-You know us as the world’s largest retailer, but we’re also one of the world’s fastest growing [e-commerce](www.walmartlabs.com) companies. [Electrode](www.github.com/electrode-io) is the platform that powers the Walmart.com customer experience. When you use [Walmart.com](www.walmart.com), you use Electrode — so you know it’s battle tested!
+We reuse React components across all of our brands, which means our developers
+need to be able to search through thousands of components, view their
+documentation and see them rendered, and use another developer's component
+secure in the knowledge that its structure and implementation are consistent
+with our standards. We needed a way to ensure that all components were built
+according to modern best practices, without slowing our developers down with a
+lot of manual configuration.
 
-We solve complex problems on an unimaginably big scale and build [awesome modules](stand_alone_modules.html) that optimize performance and efficiency. [Electrode](what is electrode.html) enforces modern web development best practices throughout the development cycle, encouraging clean, modular, reusable components while remaining easy to develop and deploy on.
+We serve millions of customers a day, and performance is vital to our business.
+React's server-side rendering has the potential to improve performance and SEO,
+but it can be complicated to set up. We needed a platform with server-side
+rendering built in by default, so new projects would be optimized from day one.
 
-Electrode also supports isomorphic rendering/universal JavaScript, CSRF (Cross Site Request Forgery) protection, internationalization, component tooling, progressive web app standards, and everything developers need to successfully develop and deploy complex enterprise-scale React/Node.js applications.
+Configuration and deployment of a web app at this scale can get out of hand
+quickly. We needed tools to manage multiple complex configurations, deploy to
+multiple environments and hosts, and optimize the bundles we deliver to
+customers.
 
-We are continuing to improve Electrode—future enhancements will include more progressive web app features for mobile, bigger investments in performance, and much more. Learn about what inspires us and [read more about](stand_alone_modules.html) the framework we are building.
+To solve these problems, we created the Electrode platform. Electrode consists
+of three pillars: Electrode Core, Electrode Modules, and Electrode Tools.
+
+*Electrode Core* provides a set of modules that get you started with a simple,
+consistent structure that follows modern best practices. When you're ready to
+take your app into production, Electrode automatically deploys to your favorite
```

Comment:
not automatically, but makes it easy.

We should mention we have docker containers ready for electrode already